core/frontend/src/main/scala/io/udash/bindings/inputs/RadioButtons.scala

```diff
@@ -32,7 +32,7 @@ object RadioButtons {
     selectedItem: Property[T], options: ReadableProperty[Seq[T]]
```

Comment:
Is there a reason to use `ReadableProperty[Seq]` here and a `SeqProperty` in the `CheckButtons`? We could probably avoid rerendering everything if these were both SeqProperties.

Reply:
`CheckButtons` uses `ReadableProperty[Seq]` too. I think that `produce` is good enough here, but we can change the argument type to `ReadableSeqProperty` to make replacement with `repeat` backward compatible.